test/templates/form-group/index.njk

```diff
@@ -27,4 +27,56 @@
   {% endset %}
   {{ format(example) }}
 
+  <h2>Inline radio</h2>
+
+  {% set example %}
+    <fieldset class="fd-fieldset">
+      <legend class="fd-fieldset__legend">Inline Radio buttons</legend>
+      <div class="fd-form-group--inline">
+          <div class="fd-form-item">
+<label class="fd-form-label fd-form-label--radio" for="pDidh767">
+                  <input type="radio" class="fd-radio" id="pDidh767" name="radio">Field label
+              </label>
+          </div>
+          <div class="fd-form-item">
+              <label class="fd-form-label fd-form-label--radio" for="pDidh7618">
+                  <input type="radio" class="fd-radio" id="pDidh7618" name="radio" >Field label
+              </label>
+          </div>
+          <div class="fd-form-item">
+              <label class="fd-form-label fd-form-label--radio" for="pDidh7619">
+                  <input type="radio" class="fd-radio" id="pDidh7619" name="radio">Field label
+              </label>
+          </div>
+      </div>
+  </fieldset>
+{% endset %}
+{{ format(example) }}
+<br>
+
+  <h2>Checkbox inline </h2>
+
+  {% set example %}
+    <fieldset class="fd-fieldset">
+        <legend class="fd-fieldset__legend">Checkboxes inline</legend>
+        <div class="fd-form-group--inline">
```

Comment:
Should be `fd-form-group fd-form-group--inline`. `fd-form-group` brings the reset